system-tests/lib/cre/don/jobs/gateway/models.go

```diff
@@ -0,0 +1,93 @@
+package gateway
+
+type GatewayJobSpec struct {
+	Type              string        `toml:"type"`
+	SchemaVersion     int           `toml:"schemaVersion"`
+	ExternalJobID     string        `toml:"externalJobID"`
+	Name              string        `toml:"name"`
+	ForwardingAllowed bool          `toml:"forwardingAllowed"`
+	GatewayConfig     GatewayConfig `toml:"gatewayConfig"`
+}
+
+type GatewayConfig struct {
+	ConnectionManagerConfig ConnectionManagerConfig `toml:"ConnectionManagerConfig"`
+	DONs                    []DON                   `toml:"Dons"`
+	NodeServerConfig        NodeServerConfig        `toml:"NodeServerConfig"`
+	UserServerConfig        UserServerConfig        `toml:"UserServerConfig"`
+	HTTPClientConfig        HTTPClientConfig        `toml:"HTTPClientConfig"`
+}
+
+type NodeServerConfig struct {
+	HandshakeTimeoutMillis int    `toml:"HandshakeTimeoutMillis"`
+	MaxRequestBytes        int    `toml:"MaxRequestBytes"`
+	Path                   string `toml:"Path"`
+	Port                   int    `toml:"Port"`
+	ReadTimeoutMillis      int    `toml:"ReadTimeoutMillis"`
+	RequestTimeoutMillis   int    `toml:"RequestTimeoutMillis"`
+	WriteTimeoutMillis     int    `toml:"WriteTimeoutMillis"`
+}
+
+type UserServerConfig struct {
+	ContentTypeHeader    string   `toml:"ContentTypeHeader"`
+	MaxRequestBytes      int      `toml:"MaxRequestBytes"`
+	Path                 string   `toml:"Path"`
+	Port                 int      `toml:"Port"`
+	ReadTimeoutMillis    int      `toml:"ReadTimeoutMillis"`
+	RequestTimeoutMillis int      `toml:"RequestTimeoutMillis"`
+	WriteTimeoutMillis   int      `toml:"WriteTimeoutMillis"`
+	CORSEnabled          bool     `toml:"CORSEnabled"`
+	CORSAllowedOrigins   []string `toml:"CORSAllowedOrigins"`
+}
+
+type HTTPClientConfig struct {
+	MaxResponseBytes int      `toml:"MaxResponseBytes"`
+	AllowedPorts     []int    `toml:"AllowedPorts,omitempty"`
+	AllowedIps       []string `toml:"AllowedIps,omitempty"`
+	AllowedIPsCIDR   []string `toml:"AllowedIPsCIDR,omitempty"`
+}
+
+type ConnectionManagerConfig struct {
+	AuthChallengeLen          int    `toml:"AuthChallengeLen"`
+	AuthGatewayId             string `toml:"AuthGatewayId"`
+	AuthTimestampToleranceSec int    `toml:"AuthTimestampToleranceSec"`
+	HeartbeatIntervalSec      int    `toml:"HeartbeatIntervalSec"`
+}
+
+type DON struct {
+	DonId         string
+	F             int
+	HandlerName   string
+	Members       []DONMember
+	HandlerConfig HandlerConfig
```

Comment:
I think HandlerConfig is different per handler. e.g. HTTP Capabilities handler config is defined [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/gateway/handlers/capabilities/v2/http_handler.go#L59)